interpreter/interpreter.py

```diff
@@ -130,6 +131,43 @@ def cli(self):
     # modifies it according to command line flags, then runs chat.
     cli(self)
 
+
+  def save_model_config(self, model_name, model_path, config_file='model_config.json'):
+      # Check if model_name and model_path are not empty
+      if not model_name or not model_path:
+          raise ValueError("Model name and model path cannot be empty.")
+      try:
+          configs = []
+
+          # Add new configuration
+          config = {
+              'model_name': model_name,
+              'model_path': model_path
+          }
+          configs.append(config)
+
+          # Save configurations
+          with open(config_file, 'w') as file:
+              json.dump({'config': configs}, file)
+      except:
+          traceback.print_exc()
+
+  def load_model_config(self, config_file='model_config.json'):
+      try:
+          if os.path.exists(config_file) and os.path.getsize(config_file) > 0:
+              with open(config_file, 'r') as file:
+                  configs = json.load(file)['config']
+                  if configs:
+                      for config in configs:
+                          if 'model_name' in config and 'model_path' in config:
+                              # Check if model_name and model_path are not empty strings
+                              if config['model_name'].strip() and config['model_path'].strip():
+                                  return config['model_name'], config['model_path']
+          return None, None
+      except:
+          return None, None
```

Comment:
would you add the error message if exception happened, otherwise the silent exception would make it difficult to debug by user.

Reply:
Okay would add error message here